source/cloud/windows.md

```diff
@@ -33,33 +38,78 @@ Click or select the `Overview` tab, you will see the Windows admin password
+:::{warning}
+Any user with access to the project can see the password in the metadata of the VM. 
+You can change the password in windows settings if necessary. Make sure to write down the password, since we cannot help you reset it if you lose it.
 :::
 
 At this point you can use Windows Remote Desktop (RDP) with your new admin user and password
 to connect to your VM.
-Make sure you use the correct port to connect to your Windows Remote Desktop,
+:::{note}
+Remember: use `terraform output` to get the connection details for your VM
 (please see [Terraform section](terraform.md) for more details).
+:::
 
 :::{note}
 Take into account you should upgrade your Windows OS after the first boot.
 You should activate your Windows machine with a valid key as well.
 :::
+# Creating your own Windows image
+It is possible to customize and upload your own windows image. 
+The guide below is a semi-automated method of doing so. 
+## Preperation
+1) Download the latest stable virtio driver:
+https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/
+3) Save as `virtio-win.iso`
+4) Download the latest windows iso, save as `Windows.iso`
+* [Win10](https://www.microsoft.com/en-us/software-download/windows10ISO)
+5) Configure/Download the unattended install iso: [https://schneegans.de/windows/unattend-generator/](https://schneegans.de/windows/unattend-generator/?LanguageMode=Unattended&UILanguage=en-US&UserLocale=nl-BE&KeyboardLayout=0409%3A00000409&GeoLocation=21&ProcessorArchitecture=amd64&BypassRequirementsCheck=true&ComputerNameMode=Custom&ComputerName=openstack&TimeZoneMode=Implicit&PartitionMode=Interactive&WindowsEditionMode=Unattended&WindowsEdition=pro&UserAccountMode=Unattended&AccountName0=Admin&AccountPassword0=&AccountGroup0=Administrators&AutoLogonMode=None&PasswordExpirationMode=Unlimited&LockoutMode=Default&EnableRemoteDesktop=true&DisableAppSuggestions=true&DisableWidgets=true&PreventDeviceEncryption=true&WifiMode=Skip&ExpressSettings=DisableAll&SystemScript0=Set-Content+-Path+C%3A%5CWindows%5Ccloudconfig.ps1+-Value+%22Add-Content+-Path+%27C%3A%5CProgram+Files%5CCloudbase+Solutions%5CCloudbase-Init%5Cconf%5Ccloudbase-init*%27+-Value+%27first_logon_behaviour%3Dno%27%22&SystemScriptType0=Ps1&SystemScript1=pnputil+-i+-a+F%3A%5Cviostor%5Cwin10%5Camd64%5C*.INF&SystemScriptType1=Cmd&SystemScript2=pnputil+-i+-a+F%3A%5Cqxl%5Cw7%5Camd64%5C*.INF&SystemScriptType2=Ps1&FirstLogonScript0=Set-ExecutionPolicy+Unrestricted&FirstLogonScriptType0=Cmd&FirstLogonScript1=Remove-Partition+-Disknumber+0+-PartitionNumber+%24%28Get-Partition+%7C+Select-Object+-Last+1%29.PartitionNumber+-Confirm%3A%24false&FirstLogonScriptType1=Ps1&FirstLogonScript2=Invoke-WebRequest+-UseBasicParsing+https%3A%2F%2Fgithub.com%2Fcloudbase%2Fcloudbase-init%2Freleases%2Fdownload%2F1.1.4%2FCloudbaseInitSetup_1_1_4_x64.msi+-OutFile+cloudbaseinit.msi%3B+cloudbaseinit.msi+&FirstLogonScriptType2=Ps1&WdacMode=Skip)(link includes current configuration)
```

Comment:
The provided link does not enables the execution of PowerShell script files, could you please change that option and update the link?